### PR TITLE
code for back office changes

### PR DESCRIPTION
**Pull request by takanocap**
## Description of changes

changes to backoffice claim 

## Checklist

- [X] Content has no typos
- [X] Correct inline validation errors
- [X] Back link and next page navigation
- [X] Removed all unnecessary console logs
- [X] Removed all commented out code
- [ ] Updated README / confluence

### CODE REVIEW

File: app/views/view-claim.njk [hunk 1]
```diff
@@ -34,8 +34,8 @@
         }) }}
       {% endif %}
 
-      <h1 class="govuk-caption-l govuk-!-margin-bottom-3">Claim number: {{reference}}</h1>
-      <h1 class="govuk-heading-l govuk-!-margin-bottom-7">{{ title }} <span class="govuk-tag {{ status.tagClass }}">{{ status.capitalisedtype }}</span></h1>
+      <h1 class="govuk-caption-l govuk-!-margin-bottom-3">Claim number by <a href="/view-agreement/{{applicationReference}}">{{ title }}</a> </h1>
```
Comment (alex742) [inline app/views/view-claim.njk:37]: It should be "Claim by" not "Claim number by"

Also I think the URL is wrong. Do you have endemics turned ON in your .env file? I think Elamin had this same issue and that fixed it. The new url is /agreement/reference/claims

File: app/views/view-claim.njk [hunk 2]
```diff
@@ -56,7 +56,7 @@
       {% endif %}
 
       {% set claimHTML %}
-        <h2 class="govuk-heading-l govuk-!-margin-bottom-5">{{ status.normalType }}</h2>
+        <h2 class="govuk-heading-l govuk-!-margin-bottom-5">Claim Details</h2>
```
Comment (alex742) [inline app/views/view-claim.njk:59]: "Claim details" not "Claim Details"

File: app/views/view-claim.njk [hunk 3]
```diff
@@ -34,8 +34,8 @@
         }) }}
       {% endif %}
 
-      <h1 class="govuk-caption-l govuk-!-margin-bottom-3">Claim number: {{reference}}</h1>
-      <h1 class="govuk-heading-l govuk-!-margin-bottom-7">{{ title }} <span class="govuk-tag {{ status.tagClass }}">{{ status.capitalisedtype }}</span></h1>
+      <h1 class="govuk-caption-l govuk-!-margin-bottom-3">Claim number by <a href="/view-agreement/{{applicationReference}}">{{ title }}</a> </h1>
+      <h1 class="govuk-heading-l govuk-!-margin-bottom-7"> Claim number: {{reference}} <span class="govuk-tag {{ status.tagClass }}">  {{ status.capitalisedtype }}</span></h1>
```
Comment (alex742) [inline app/views/view-claim.njk:38]: Do we need the space before "  {{ status.capitalisedtype }"?

File: app/routes/view-claim.js [hunk 4]
```diff
@@ -127,6 +130,8 @@ module.exports = {
           })
         : [])
 
+      const status = { key: { text: 'Status' }, value: { html: `<span class='govuk-tag ${getStyleClassByStatus(formatStatusId(statusId))}'> ${upperFirstLetter(claimStatus?.status)} </span>` } }
+      const ClaimDate = { key: { text: 'Claim date' }, value: { html: convertDateToFormattedString(createdAt) } }
```
Comment (alex742) [inline app/routes/view-claim.js:134]: Why is this capitalised? Should be "claimDate" not "ClaimDate"

File: app/routes/view-claim.js [hunk 5]
```diff
@@ -161,6 +166,8 @@ module.exports = {
       }
 
       const beefRows = [
+        status,
+        ClaimDate,
```
Comment (alex742) [inline app/routes/view-claim.js:170]: claimDate

File: app/routes/view-claim.js [hunk 6]
```diff
@@ -180,6 +187,8 @@ module.exports = {
       ]
 
       const dairyRows = [
+        status,
+        ClaimDate,
```
Comment (alex742) [inline app/routes/view-claim.js:191]: claimDate

File: app/routes/view-claim.js [hunk 7]
```diff
@@ -199,6 +208,8 @@ module.exports = {
       ]
 
       const sheepRows = [
+        status,
+        ClaimDate,
```
Comment (alex742) [inline app/routes/view-claim.js:212]: claimDate

File: app/routes/view-claim.js [hunk 8]
```diff
@@ -220,6 +231,8 @@ module.exports = {
       ]
 
       const pigRows = [
+        status,
+        ClaimDate,
```
Comment (alex742) [inline app/routes/view-claim.js:235]: claimDate

File: app/views/view-claim.njk (hunk 3 above)
Comment (takanocap) [reply app/views/view-claim.njk:38]: We do but I can remove the space

File: app/routes/view-claim.js (hunk 4 above)
Comment (takanocap) [reply app/routes/view-claim.js]: This reference is a claim reference from my test

File: app/views/view-claim.njk (hunk 1 above)
Comment (takanocap) [reply app/views/view-claim.njk]: but reference is Claim reference will that work for agreement ?

**Review by alex742 — APPROVED** [review]

**Review by alex742 — APPROVED** [review]

**Pull request merged**